client/my-sites/plugins/main.jsx

```diff
@@ -466,7 +466,7 @@ export class PluginsMain extends Component {
 			<>
 				<DocumentHead title={ pageTitle } />
 				<QueryPlugins siteId={ selectedSite?.ID } />
-				<QuerySiteFeatures siteIds={ this.props.siteIds } />
+				<QueryJetpackSitesFeatures />
```

Comment:
`QuerySiteFeatures` made a network call once per site, `QueryJetpackSitesFeatures` calls only once once.

Reply:
Good call, I've updated it to use the single site endpoint.